doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,389 @@
+.. _bluetooth_shell:
+
+Bluetooth Shell
+###############
+
+The Bluetooth Shell is an application based on the :ref:`shell_api` module. It offer a collection of
+commands made to easily interact with the Bluetooth stack.
+
+Bluetooth Shell Setup and Usage
+*******************************
+
+First you need to build and flash your board with the Bluetooth shell. For how to do that, see the
+:ref:`getting_started`. The Bluetooth shell itself is located in
+:zephyr_file:`tests/bluetooth/shell/`.
+
+When it's done, connect to the CLI using your favorite serial terminal application. You should see
+the following prompt:
+
+.. code-block:: console
+
+        uart:~$
+
+For more details on general usage of the shell, see :ref:`shell_api`.
+
+The first step is enabling Bluetooth. To do so, use the :code:`bt init` command. Messages should be
+printed confirming Bluetooth has been initialized. The following message is printed to confirm
+Bluetooth has been initialized.
```

Comment:
There's now two sentences saying the same thing.

Reply:
Oh yeah oops, sorry 😅